x/auth/spec/05_vesting.md

```diff
@@ -88,49 +88,64 @@ type VestingAccount interface {
   GetStartTime() int64
   GetEndTime()   int64
 }
+```
 
+```proto
 // BaseVestingAccount implements the VestingAccount interface. It contains all
 // the necessary fields needed for any vesting account implementation.
-type BaseVestingAccount struct {
-  BaseAccount
+message BaseVestingAccount {
```

Comment:
could we use `+++` here (see https://github.com/cosmos/cosmos-sdk/pull/8279#discussion_r553882601)